bot.py

```diff
@@ -31,6 +37,84 @@
 lottery = Lottery(time_delta=givenTime)
 
 
+async def time_left(idLottery):
+    print(f"what is timeleft?")
+    async with aiohttp.ClientSession() as session:
+        async with session.get(f"{DATABASE_URL}/lottery/timeLeft?idLottery={idLottery}") as response:
+            data = await response.json()  # Parse the response JSON
+            print(f"timeLeft is {data}")
+            return data
+
+            # if response.status == 200:
+            #     data = await response.json()  # Parse the response JSON
+            #     print(data)
+            #     return data
+            # else:
+            #     return 'Error: ' + str(response.status)
+
+
+async def winning_fruit(idLottery):
+    async with aiohttp.ClientSession() as session:
+        async with session.get(f"{DATABASE_URL}/lottery/winningFruit?idLottery={idLottery}") as response:
+            data = await response.json()  # Parse the response JSON
+            print(data)
+            return data
+
+
+async def start_lottery():
+    data = None
+    async with aiohttp.ClientSession() as session:
+        async with session.post(f"{DATABASE_URL}/lottery") as response:
+            data = await response.json()
+            print(data)
+
+
+async def get_winners(idLottery):
+    data = None
+    async with aiohttp.ClientSession() as session:
+        async with session.get(f"{DATABASE_URL}/lottery/winners?idLottery={idLottery}") as response:
+            data = await response.json()
+            print(data)
+
+
+async def get_id_lottery():
+    data = None
+    async with aiohttp.ClientSession() as session:
+        async with session.get(f"{DATABASE_URL}/lottery/running") as response:
+            data = await response.json()
+            print(data)
+            return data
+
+
+async def stop_lottery():
+    data = None
+    async with aiohttp.ClientSession() as session:
+        async with session.post(f"{DATABASE_URL}/lottery/stop") as response:
+            data = await response.json()
+            print(data)
+
+
+async def post_bet(idUser, idLottery, userBet):
+    data = None
+    vote = {
+        "idUser": idUser,
+        "idLottery":  idLottery,
+        "userBet": userBet
+    }
+
+    async with aiohttp.ClientSession() as session:
+        async with session.post(f"{DATABASE_URL}/users_vote", json=vote) as response:
```

Comment:
Endpoints are specific, they are not really related to codestyle of Python. These are general links. Using symbols like `_` is generally not welcomed because they are making hard reading URI. Preferred way sometimes introducing new paths like `/users/vote` assuming that there are many different endpoints under `users`.

Reply:
I see, I'll look into that and change it

Reply:
introduced new paths and made them camelcase